yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -436,17 +448,17 @@ func checkAndHandleSnapshotComplete(status *dbzm.ExportStatus, progressTracker *
 		return false, nil
 	}
 	progressTracker.Done(status)
-	setDataIsExported()
-	err := writeDataFileDescriptor(exportDir, status)
-	if err != nil {
-		return false, fmt.Errorf("failed to write data file descriptor: %w", err)
-	}
-	log.Infof("snapshot export is complete.")
-	err = renameDbzmExportedDataFiles()
-	if err != nil {
-		return false, fmt.Errorf("failed to rename dbzm exported data files: %v", err)
-	}
 	if !isTargetDBExporter(exporterRole) {
+		setDataIsExported()
```

Comment:
FYI while merging: I added a condition here to check if config.SnapshotMode != "never" (need this even in case of sourceDBExporter beccause we now use pg_dump. so can't use `isTargetDBExporter`)